staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -1134,7 +1207,7 @@ func TestValidationExpressions(t *testing.T) {
 					"9":   2,
 					"int": 3,
 					"./-": 4,
-					"👑":   5,
+					"👑":  5,
```

Comment:
weird that gofmt was happy with both of these

Reply:
maybe something along the lines of https://github.com/golang/go/issues/7481 ?

Reply:
My editor (IntelliJ Golang) can't seem to agree with previous versions of itself on what the correct rune with is..

Reply:
Oh nvm. gofmt is failing on this file. It's purely a problem with my editor.

Reply:
> weird that gofmt was happy with both of these

Narrator: "it wasn't"

```diff
-					"👑":  5,
+					"👑":   5,
 				},
 			},
 			schema: objectTypePtr(map[string]schema.Structural{"m": mapType(&integerType)}),

Run ./hack/update-gofmt.sh
```